java-frontend/src/main/java/org/sonar/java/se/xproc/MethodYield.java

```diff
@@ -158,7 +158,7 @@ public boolean equals(Object obj) {
       .isEquals();
   }
 
-  static Stream<Constraint> pmapToStream(@Nullable  PMap<Class<? extends Constraint>, Constraint> pmap) {
+  public static Stream<Constraint> pmapToStream(@Nullable  PMap<Class<? extends Constraint>, Constraint> pmap) {
```

Comment:
This one should also probably not have an extended visibility but rather be reimplemented.